lms/static/certificates/sass/_utilities.scss

```diff
@@ -70,3 +70,14 @@ $font-family-serif: 'Domine', serif;
     border-bottom-width: rem(2);
     border-bottom-style: solid;
 }
+
```

Comment:
Per [my previous comment](https://github.com/edx/edx-platform/pull/8139/files#r31039536), Please remove this addition.